Добавлены енам мессенджер и роль

TechMePLZ opened the pull request:
добавлены комментарии
Обозначены аннотации на не обновляемые поля таблиц

TechMePLZ committed:
Добавлены енам мессенджер и рольб
добавлены комментарии
Обозначены аннотации на не обновляемые поля таблиц

ScriptingGuyRU commented on src/main/java/ru/friend/crmcs/entity/OrderCS.java:
```diff
@@ -21,21 +21,43 @@ public class OrderCS {
+     * true = задача выполнена, false = задача не выаолнена
+     * Значение по умолчанию = false
      */
     private Boolean isReady;
```
Лучше использовать примитив, а не обертку

ScriptingGuyRU commented on src/main/java/ru/friend/crmcs/entity/OrderCS.java:
```diff
@@ -21,21 +21,43 @@ public class OrderCS {
+     * true = задача закрыта, false = задача не закрыта
+     * Значение по умолчанию = false
+     */
     private Boolean isActive;
```
Где прописывается "Значение по умолчанию"? В коде не вижу такого

ScriptingGuyRU commented on src/main/java/ru/friend/crmcs/entity/UserMember.java:
```diff
@@ -17,13 +17,26 @@ public class UserMember {
     private String username;
 
+    /**
+     * пасворд пользователя системы (занчение зашивровано Base64)
```
Лучше "Пароль", а не "Пассворд"

ScriptingGuyRU commented on src/main/java/ru/friend/crmcs/entity/Client.java:
```diff
@@ -1,2 +1,62 @@
+
+    /**
+     * Сторковое представление основного контактного номера телефона
+     * !!!(не может быть null)
```
Минор: !!!(не может быть null) можно  не писать, по аннотации nullable это понятно

ScriptingGuyRU commented on src/test/java/ru/friend/crmcs/CrmcsApplicationTests.java:
```diff
@@ -6,8 +6,8 @@
-	@Test
-	void contextLoads() {
-	}
+    @Test
```
Зачем пустой тест?

ScriptingGuyRU reviewed (commented):
Добавил замечания

TechMePLZ committed:
Перемещены все сущности в отдельные пакеты.
Выставлены значение по умолчанию для булевых полей классов.
Поправлены комментарии
